maintainers/maintainer-list.nix

```diff
@@ -16186,4 +16186,10 @@
     github = "ziguana";
     githubId = 45833444;
   };
+  farcaller = {
```

Comment:
We try to keep the listing alphabetically.

Reply:
Fixed! I noticed it wasn't the case in the bottom of the list so wasn't sure. Thanks for correcting me.

Reply:
For time to time the list is manually sorted as it happens that entries are sneaking in at the wrong position.

Thanks for moving your entry.